packages/letters-of-credit/src/components/Pages/EllaPage/EllaPage.js

```diff
@@ -134,9 +134,13 @@ class EllaPage extends Component {
       return (
```

Comment:
We're just missing a few of the styling changes for Ella's page to totally match the hi-fis, mostly to do with the table display. The most recent hifi on box looks like this:

![4 - ellahome](https://user-images.githubusercontent.com/35696678/39421957-4d965d52-4c63-11e8-95c1-94a8dc88b883.png)